core/src/main/scala/kafka/server/ZkAdminManager.scala

```diff
@@ -118,7 +118,7 @@ class ZkAdminManager(val config: KafkaConfig,
     metadataAndConfigs.get(topicName).foreach { result =>
       val logConfig = LogConfig.fromProps(LogConfig.extractLogConfigMap(config), configs)
       val createEntry = configHelper.createTopicConfigEntry(logConfig, configs, includeSynonyms = false, includeDocumentation = false)(_, _)
-      val topicConfigs = logConfig.values.asScala.map { case (k, v) =>
+      val topicConfigs = logConfig.nonInternalValues.asScala.map { case (k, v) =>
```

Comment:
Do we need to fix any test related to this change?

Reply:
No tests need to be fixed.

The test that covers this code is `testCreateTopicsResponseMetadataAndConfig`. The test makes sure the returned values of CreateTopic and DescribeTopic are the same. Previously, the behavior of CreateTopic and DescribeTopic are different for internal configs, but the test passed because we don't have internal configs in LogConfig. Now, we have internal configs and this test failed, then I changed this line to let the test pass: CreateTopic and DescribeTopic both don't return internal configs.